js/page.js

```diff
@@ -0,0 +1,30 @@
+(function () {
```

Comment:
We should probably leave a one-line comment above this function saying what it does, e.g.
```js
// Adds an "Open in New Tab" link to /page pages which simply embed something.
(function () {
...
```

Especially since we have early returns from this function, so we wouldn't want to inadvertently add unrelated changes in the same anonymous function.